docs/examples/timer_operations.rst

```diff
@@ -15,33 +14,57 @@ define our recurring Timer job.
         NativeAppAuthClient,
         TimerClient,
         TimerJob,
-        TransferClient,
         TransferData,
     )
+    from globus_sdk.scopes import (
+        GCSCollectionScopeBuilder,
+        MutableScope,
+        TimerScopes,
+        TransferScopes,
+    )
 
     GOEP1 = "ddb59aef-6d04-11e5-ba46-22000b92c6ec"
     GOEP2 = "ddb59af0-6d04-11e5-ba46-22000b92c6ec"
+    NATIVE_CLIENT_ID = "61338d24-54d5-408f-a10d-66c06b59f6d2"
     TIMER_CLIENT_ID = "524230d7-ea86-4a52-8312-86065a9e0417"
-    TIMER_SCOPE = f"https://auth.globus.org/scopes/{TIMER_CLIENT_ID}/timer"
-    TRANSFER_SCOPE = "urn:globus:auth:scope:transfer.api.globus.org:all"
-    SCOPES = [TIMER_SCOPE, TRANSFER_SCOPE]
+
+    # If any of the collections you're targeting are mapped collections
+    # that require data_access scopes, include them in this list
+    MAPPED_COLLECTION_IDS = []
+
+    # Build a scope that will give the Timer service
+    # access to perform transfers on your behalf
+    timer_scope = TimerScopes.make_mutable("timer")
+    transfer_scope = TransferScopes.make_mutable("all")
+    transfer_action_provider_scope = MutableScope(
+        "https://auth.globus.org/scopes/actions.globus.org/transfer/transfer"
+    )
+
+    # If you declared and mapped collections above, add them to the transfer scope
+    if MAPPED_COLLECTION_IDS:
+        for id in MAPPED_COLLECTION_IDS:
+            gcs_data_access_scope = GCSCollectionScopeBuilder(id).make_mutable(
+                "data_access",
+                optional=True,
+            )
+            transfer_scope.add_dependency(gcs_data_access_scope)
+        transfer_action_provider_scope.add_dependency(transfer_scope)
+
+    timer_scope.add_dependency(transfer_action_provider_scope)
+
+    print(f"Requesting scopes: {timer_scope}")
+
+    # Initialize your native app auth client
+    native_client = NativeAppAuthClient(NATIVE_CLIENT_ID)
 
     # Get access tokens to use for both Transfer and Timer
-    native_client.oauth2_start_flow(requested_scopes=SCOPES)
+    native_client.oauth2_start_flow(requested_scopes=timer_scope)
     authorize_url = native_client.oauth2_get_authorize_url()
     print(f"Please go to this URL and login:\n\n{authorize_url}\n")
     auth_code = input("Enter the auth code here: ").strip()
+
     token_response = native_client.oauth2_exchange_code_for_tokens(auth_code)
     timer_token = token_response.by_resource_server[TIMER_CLIENT_ID]["access_token"]
-    transfer_token = token_response.by_resource_server["transfer.api.globus.org"][
-        "access_token"
-    ]
-
-    # Set up a transfer client and create a `TransferData` object
-    transfer_authorizer = AccessTokenAuthorizer(transfer_token)
-    transfer_client = TransferClient(authorizer=transfer_authorizer)
-    data = TransferData(transfer_client, GOEP1, GOEP2)
-    data.add_item("/share/godata/file1.txt", "/~/file1.txt")
```

Comment:
I'm seeing this removal, but not the redefinition of `data`. Do you have an unpushed change for this, perhaps?

I'm guessing that we just want
```python
data = TransferData(source_endpoint=GOEP1, destination_endpoint=GOEP2)
data.add_item(...)
```

Reply:
Yep—this was missed in a a copy-paste error when I was moving bits from the script I wrote for the support ticket! 🤦‍♀️